<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileEditChild</class>
 <widget class="QWidget" name="FileEditChild">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileEditChild</string>
  </property>
  <widget class="QPushButton" name="saveBut">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>540</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editBut">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>450</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="editPreviewText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1041</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveasBut">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>630</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delBut">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>360</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="textpreviewScrollBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>1041</width>
     <height>26</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="typeTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>821</width>
     <height>461</height>
    </rect>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
